pkg/features/list.go

```diff
@@ -126,4 +126,7 @@ var (
 
 	// Display RHSA/RHBA/RHEA advisory separately from associated CVE.
 	CVEAdvisorySeparation = registerFeature("Display RHSA/RHBA/RHEA advisory separately from associated CVE", "ROX_CVE_ADVISORY_SEPARATION")
+
+	// Add the ability to generate an SBOM from an image
+	SBOMGeneration = registerFeature("Add the ability to generate an SBOM from an image", "ROX_SBOM_GENERATION")
```

Comment:
In the future we will likely have SBOM consumption as well - from a back-end perspective I don't believe we'll _need_ another feature flag for that - do you foresee it being beneficial to have two feature flags (Generation + Consumption)? If so then LGTM - if not then perhaps just `ROX_SBOM`? (or similar?) Thoughts?

Reply:
Mark is faster than I am, so +1 to everything he said. :)

Additionally, if SBOM generation is stable enough well before the release I think we can delete this flag entirely instead of shipping with it enabled. In that case, a back-end flag for consumption would be (or could be) totally independent.